### PR TITLE
Fixed missing project_id issue

### LINKED ISSUE #4: [BUG] ActiveRecord::RecordNotFound (Couldn't find Project with identifier=undefined):
**Describe the bug**

```
F, [2019-08-23T21:05:04.528924 #21015] FATAL -- :   
F, [2019-08-23T21:05:04.529149 #21015] FATAL -- : ActiveRecord::RecordNotFound (Couldn't find Project with identifier=undefined):
F, [2019-08-23T21:05:04.529199 #21015] FATAL -- :   
F, [2019-08-23T21:05:04.529262 #21015] FATAL -- : app/models/project.rb:316:in `find'
plugins/redmine_text_blocks/app/controllers/text_blocks_controller.rb:81:in `find_project_by_project_id'
```

`undefined` cause may be client side.

### PR DESCRIPTION
**Pull request by sanak**
Fixes #4.

Changes proposed in this pull request:
- Add `text_block_project_id` as hidden input tag ~~when~~ to support available project is only one case.

@gtt-project/maintainer

### CODE REVIEW

**Review by dkastl — APPROVED** [review]

**Comment by sanak** [comment]
@dkastl Thanks for approving this!
I will merge this now. (CC: @mopinfish)

**Comment by mopinfish** [comment]
@sanak I've checked this changes. LGTM.

**Comment by sanak** [comment]
@mopinfish Okay, thanks!

File: assets/javascripts/text_blocks.js [hunk 1]
```diff
@@ -35,9 +35,16 @@ var TextBlocks = {
   },
 
   reload: function(e){
+    debugger
```
Comment (sanak) [inline assets/javascripts/text_blocks.js:38]: @dkastl @mopinfish  
Sorry, I forgot to delete this line...
I will delete this line on `main` branch directly.

Comment (sanak) [reply assets/javascripts/text_blocks.js:38]: Well, it seems to impossible to push `main` branch directly, so I will make PR to delete this line...

**Pull request merged**